nvidia-php8.1-fpm-ffmpeg-bullseye/Dockerfile

```diff
@@ -60,12 +60,20 @@ FROM base as ffmpeg-builder
 RUN apt -y install build-essential cmake gcc git libc6 libc6-dev libtool nasm nvidia-cuda-toolkit yasm
 # Ffmpeg
 WORKDIR /src
+# Install libaom from source.
+RUN mkdir -p ~/ffmpeg_sources/libaom && \
+  cd ~/ffmpeg_sources/libaom && \
+  git clone https://aomedia.googlesource.com/aom && \
+  cmake ./aom && \
+  make && \
+  sudo make install
```

Comment:
There's no reason to make a random directory like this.  PWD is already /src.  See how nv-codec-headers is built.

```
RUN git clone https://aomedia.googlesource.com/aom && cd aom && \
      cmake . && make && make install
```

Reply:
seen that, i have updated the installion path